src/backend/expungeservice/models/record_summary.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass, field
+from typing import List, Dict
+
+from expungeservice.models.case import Case
+from expungeservice.models.charge import Charge
+
+
+@dataclass
+class RecordSummary:
+    total_charges: int
+    cases_sorted: Dict[str, List[str]]
+    eligible_charges: List[str]
+    county_balances: Dict[str, float]
+
+    @property
+    def total_balance_due(self):
+        total_in_cents = 0
+        for county in self.county_balances.keys():
+            total_in_cents += round(self.county_balances[county] * 100)
+        return round(total_in_cents / 100.0, 2)
```

Comment:
Note that at https://github.com/codeforpdx/recordexpungPDX/pull/721/files#diff-739da94a6cb453db19351da6939fa0d5R20, you're adding floats. I've had this discussion with @NickSchimek before but since we're adding floats with 2 decimal places, we will never lose accuracy to the point it matters. The increase in complexity for doing this "multiply, add, and divide/round routine" is probably more likely to cause bugs.